examples/flutter_gallery/lib/demo/cupertino/cupertino_switch_demo.dart

```diff
@@ -15,15 +15,16 @@ class CupertinoSwitchDemo extends StatefulWidget {
 }
 
 class _CupertinoSwitchDemoState extends State<CupertinoSwitchDemo> {
-
   bool _switchValue = false;
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         title: const Text('Cupertino Switch'),
-        actions: <Widget>[MaterialDemoDocumentationButton(CupertinoSwitchDemo.routeName)],
+        actions: <Widget>[
+          MaterialDemoDocumentationButton(CupertinoSwitchDemo.routeName)
+        ],
```

Comment:
this seems reasonable (though i think the original is no worse)

Reply:
fwiw, this was a bug. it is now back to the original.